keps/sig-api-machinery/20190425-crd-conversion-webhook.md

```diff
@@ -736,7 +736,11 @@ Consider including folks that also work outside the SIG or subproject.
 
 ## Graduation Criteria
 
+v1beta1:
 
+- ConversionReview API v1 is stable (including fields that need to be updated for v1beta1 or v1)
+- Conversion webhooks have comprehensive metrics
+- Step-by-step CRD version migration workflow is documented on kubernetes/website
```

Comment:
currently the [website documentation](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning/#webhook-conversion) mentions how to set up a webhook server and configure CRD to use webhook, but misses details and could benefit from examples

Reply:
The idea is that we're documenting now just how to create a converter webhook, but how to introduce a new CRD version, add the converter, migrate clients, migrate to new storage version, all the way to removing the old version. I'll rephrase this.